github-swagger.json

```diff
@@ -1050,7 +1050,9 @@
         "id",
         "login",
         "avatar_url",
-        "description"
+        "description",
+        "name",
+        "website"
```

Comment:
https://api.github.com/orgs/socialconnect

![image](https://user-images.githubusercontent.com/572096/30390210-424584a4-98f0-11e7-8f1c-0525706c0600.png)

Looks like, website doesnot exist, there is `blog` field

Reply:
Sorry about thath: I mixed up with GraphQL :(